srcpkgs/libcgroup/template

```diff
@@ -1,18 +1,16 @@
 # Template file for 'libcgroup'
 pkgname=libcgroup
-version=0.41
-revision=8
+version=2.0
+revision=1
 build_style=gnu-configure
-configure_args="--sbindir=/usr/bin --enable-pam-module-dir=/usr/lib/security"
 hostmakedepends="flex automake libtool"
 makedepends="pam-devel"
 short_desc="Library that abstracts the control group file system in Linux"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1"
-homepage="http://libcg.sourceforge.net"
-distfiles="${SOURCEFORGE_SITE}/libcg/${pkgname}-${version}.tar.bz2"
-checksum=e4e38bdc7ef70645ce33740ddcca051248d56b53283c0dc6d404e17706f6fb51
-
+homepage="https://github.com/${pkgname}/${pkgname}"
+distfiles="${homepage}/releases/download/v2.0/${pkgname}-${version}.tar.bz2"
```

Comment:
I wouldn't use $homepage and $pkgname. 2.0 should be replaced with $version. tar.gz is prefered over bz2.

Reply:
i kept $homepage and $pkgname because it shortens the url quite a bit

Reply:
I don't think we have a shortage of bytes.

Reply:
it would be nice to keep the 80 column mark as close as possible.

Reply:
distfiles often go over that mark and that's fine.

Reply:
distfile now contains full url